Objective-C/CBLDatabase.mm

```diff
@@ -327,6 +334,35 @@ - (BOOL) purgeDocumentWithID: (NSString*)documentID error: (NSError**)error {
     }
 }
 
+#pragma mark - Blob Save/Get
+
+- (BOOL) saveBlob: (CBLBlob*)blob error: (NSError**)error {
+    return [blob installInDatabase: self error: error];
+}
+
+- (CBLBlob*) getBlob: (NSDictionary*)dict {
```

Comment:
Discuss: Should this be getBlobWithProperties for Objective-C? What is your opinion?

Reply:
`getBlob: (NSDictionary*)properties ` should be fine, argument placeholder makes it clear right?

Reply:
ok